### PR TITLE
Template 2.9

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

I updated the template and tested the pipeline. I can be merged!

### CODE REVIEW

**Comment by erikrikarddaniel** [comment]
Many tests fail. Yesterday I think I saw that they plan a hotfix release (i.e. 2.9.1) to deal with urgent problems in the new release. Could it be this that cause the problems and you should wait until 2.9.1 is out?

**Comment by danilodileo** [comment]
It seems the module fastq_cat failed and I forgot to run prettier. the first one is odd as I didn't touch that module while updating the template. 

And it doesn't fail when I run the test

**Review by erikrikarddaniel — COMMENTED** [review]
How is this looking now, did you manage to fix the `fastq_cat` thing?

**Comment by danilodileo** [comment]
I fixed the issue. we should merge it @erikrikarddaniel

File: nextflow.config [hunk 1]
```diff
@@ -10,8 +10,9 @@
 params {
 
     // Input options
-    input                       = null
+    input                       = null
```
Comment (erikrikarddaniel) [inline nextflow.config]:
```suggestion
    input                       = null
```

File: nextflow.config [hunk 2]
```diff
@@ -70,7 +71,7 @@ params {
     skip_eukulele               = false
     eukulele_dbpath             = './eukulele'
     eukulele_method             = 'mets'
-
+
```
Comment (erikrikarddaniel) [inline nextflow.config]:
```suggestion

```

**Review by erikrikarddaniel — APPROVED** [review]
Looks fine, but I noticed prettier still fails. (I made two suggestions to remove trailing whitespace, maybe these were the prettier fails.)

**Pull request merged**